app/src/main/java/com/android/wardrobeManager/ClosetActivity.java

```diff
@@ -1,31 +1,78 @@
 package com.android.wardrobeManager;
 
 import android.content.Intent;
+import android.opengl.Visibility;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
 import android.view.View;
 import android.widget.Button;
+import android.widget.TextView;
 
 public class ClosetActivity extends AppCompatActivity {
 
+
+    boolean shirtsScrollVisible = false;
+    boolean shoesScrollVisible = false;
+    boolean shortsScrollVisible = false;
+
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_closet);
 
-        configureBackButton();
+    }
+
+    public void backToMainScreen(View view) {
+        startActivity(new Intent(ClosetActivity.this, StartActivity.class));
+    }
+
+    public void toggleShirtVisibility(View view) {
+
+        if (shirtsScrollVisible) {
+            shirtsScrollVisible = false;
+            findViewById(R.id.shirtScrollView).setVisibility(View.GONE);
+            ((TextView)findViewById(R.id.shirtsToggleView)).setText(">");
```

Comment:
I think these characters should be switched. The 'v' button should set the scroller to visible and the '>' button should hide it. In other words, the opposite of what you have here.